CMakeLists.txt

```diff
@@ -112,12 +121,14 @@ add_executable(create_pose_and_show src/create_pose_and_show.cpp)
 add_dependencies(create_pose_and_show ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
 target_link_libraries(create_pose_and_show sphere_discretization kinematics ${catkin_LIBRARIES})
 
-add_executable(create_sphere_and_arrow src/create_sphere_and_arrow.cpp)
-add_dependencies(create_sphere_and_arrow ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
-target_link_libraries(create_sphere_and_arrow sphere_discretization kinematics ${catkin_LIBRARIES})
+#add_executable(create_sphere_and_arrow src/create_sphere_and_arrow.cpp)
+#add_dependencies(create_sphere_and_arrow ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
+#target_link_libraries(create_sphere_and_arrow sphere_discretization kinematics ${catkin_LIBRARIES})
```

Comment:
Why were these lines commented out?  Is this no longer used?  If so, then just remove it.